addon-test-support/-private/weight-test-modules.js

```diff
@@ -0,0 +1,37 @@
+const WEIGHT_MAP = {jshint:1, eslint:1, unit:10, integration:20, acceptance:150};
```

Comment:
Can we change this object into a more readable format? Also, do we need `jshint`?

Reply:
Also, how about we call this `TEST_TYPE_WEIGHT`? It's not really relevant that it's a map, is it?

Reply:
why do you think we don't need `jshint`?

Reply:
Why do you think we do? Is this for backwards compatibility for older blueprints?